test/converted/smith-et-al.apa

```diff
@@ -0,0 +1 @@
+Smith, A. M., Katz, D. S., Niemeyer, K. E., & FORCE11 Software Citation Working Group (2016). Software citation principles. PeerJ Computer Science, 2(123), e86. https://doi.org/10.7717/peerj-cs.86
```

Comment:
This is how I *think* the APA formatted citation should look for v7 of APA.

Reply:
The other APA examples we have, have a `.` after the authors so I've added that in the hope it's correct 😬